symphony-ws/src/main/java/se/havochvatten/symphony/web/UserREST.java

```diff
@@ -188,22 +191,53 @@ public Response updateUserDefinedArea(@Context HttpServletRequest req, @PathPara
         if (req.getUserPrincipal() == null)
             throw new NotAuthorizedException("Null principal");
 
-        userDefinedAreaDto = userDefinedAreaService.updateUserDefinedArea(req.getUserPrincipal(),
+        userDefinedAreaDto = userService.updateUserDefinedArea(req.getUserPrincipal(),
                 userDefinedAreaDto);
         return Response.ok(userDefinedAreaDto).build();
     }
 
     @DELETE
     @ApiOperation(value = "Delete user defined area", response = Response.class)
-    @Path("{id}")
+    @Path("/area/{id}")
     @Produces(MediaType.APPLICATION_JSON)
     @RolesAllowed("GRP_SYMPHONY")
     public Response deleteUserDefinedArea(@Context HttpServletRequest req, @PathParam("id") Integer id) throws SymphonyStandardAppException {
         if (req.getUserPrincipal() == null)
             throw new NotAuthorizedException("Null principal");
 
-        userDefinedAreaService.delete(req.getUserPrincipal(), id);
+        userService.delete(req.getUserPrincipal(), id);
         return Response.ok().build();
     }
 
+//    @GET
```

Comment:
This whole method is commented out. Can it be removed?

Reply:
Yes, looks like a residual from development. You're right, it should be removed 👍